partner_compassion/migrations/12.0.1.1.4/post-migration.py

```diff
@@ -0,0 +1,37 @@
+import logging
+from openupgradelib import openupgrade
+
+_logger = logging.getLogger(__name__)
+
+
+@openupgrade.migrate()
+def migrate(env, installed_version):
+    if not installed_version:
+        return
+
+    def get_partners_to_change():
+        all_companies_refs = env["res.partner"].search([("is_company", "=", True)]).mapped("ref")
+        return env["res.partner"].search([("is_company", "=", False), ("ref", "in", all_companies_refs)])
+
+    partner_to_change = get_partners_to_change()
+    sponsor_to_change = partner_to_change.filtered(lambda p: any(p.mapped("sponsorship_ids.is_active")))
```

Comment:
I wonder if this works as well?
```suggestion
    sponsor_to_change = partner_to_change.filtered("sponsorship_ids.is_active")
```

Reply:
it does !